Polygon_mesh_processing/include/CGAL/Polygon_mesh_processing/distance.h

```diff
@@ -1310,8 +1322,1366 @@ double approximate_symmetric_Hausdorff_distance(const TriangleMesh& tm1,
+      recursive_hausdorff_subdivision<Kernel>(v1, v01, v12, tm2_tree, squared_error_bound)),
+    (CGAL::max)(
+      recursive_hausdorff_subdivision<Kernel>(v2 , v02, v12, tm2_tree, squared_error_bound),
+      recursive_hausdorff_subdivision<Kernel>(v01, v02, v12, tm2_tree, squared_error_bound)));
+}
+
+template< class Concurrency_tag,
+          class Kernel,
+          class TriangleMesh1,
+          class TriangleMesh2,
+          class VPM1,
+          class VPM2 >
+double bounded_error_Hausdorff_naive_impl(
+  const TriangleMesh1& tm1,
+  const TriangleMesh2& tm2,
+  const typename Kernel::FT error_bound,
+  const VPM1& vpm1,
+  const VPM2& vpm2)
+{
+  using FT         = typename Kernel::FT;
+  using Point_3    = typename Kernel::Point_3;
+  using Triangle_3 = typename Kernel::Triangle_3;
+
+  using TM2_primitive = AABB_face_graph_triangle_primitive<TriangleMesh2, VPM2>;
+  using TM2_traits    = AABB_traits<Kernel, TM2_primitive>;
+  using TM2_tree      = AABB_tree<TM2_traits>;
+
+  using TM1_face_to_triangle_map = Triangle_from_face_descriptor_map<TriangleMesh1, VPM1>;
+
+  // Initially, no lower bound is known.
+  FT squared_lower_bound = FT(0);
+
+  // Work with squares in the following, only draw sqrt at the very end.
+  const FT squared_error_bound = error_bound * error_bound;
+
+  // Build an AABB tree on tm2.
+  TM2_tree tm2_tree(faces(tm2).begin(), faces(tm2).end(), tm2, vpm2);
+  tm2_tree.build();
+  tm2_tree.accelerate_distance_queries();
+
+  // Build a map to obtain actual triangles from the face descriptors of tm1.
+  const TM1_face_to_triangle_map face_to_triangle_map(&tm1, vpm1);
+
+  // Iterate over the faces of TM1.
+  for (const auto& face : faces(tm1)) {
+
+    // Get the vertices of the face and pass them on to a recursive method.
+    const Triangle_3 triangle = get(face_to_triangle_map, face);
+    const Point_3 v0 = triangle.vertex(0);
+    const Point_3 v1 = triangle.vertex(1);
+    const Point_3 v2 = triangle.vertex(2);
+
+    // Recursively process the current triangle to obtain a lower bound on its Hausdorff distance.
+    const FT triangle_bound = recursive_hausdorff_subdivision<Kernel>(
+      v0, v1, v2, tm2_tree, squared_error_bound);
+
+    // Store the largest lower bound.
+    if (triangle_bound > squared_lower_bound) {
+      squared_lower_bound = triangle_bound;
+    }
+  }
+
+  // Return linear interpolation between found upper and lower bound.
+  return CGAL::sqrt(CGAL::to_double(squared_lower_bound));
+}
+
+} // end of namespace internal
+
+/**
+ * \ingroup PMP_distance_grp
+ * returns an estimate on the Hausdorff distance between `tm1` and `tm2` that
+ * is at most `error_bound` away from the actual Hausdorff distance between
+ * the two given meshes.
+ *
+ * @tparam Concurrency_tag enables sequential versus parallel algorithm.
+ *                         Possible values are `Sequential_tag` and `Parallel_tag`.
+ *                         Currently, parallel computation is not implemented, though.
```

Comment:
Should be clear what it means, that is the sequential version is always used (I guess)

Reply:
I rephrased a bit.